Paavalan08/paavalan
# Redesign of Personal Website Frontend

**Description:**

This PR introduces a complete redesign of the frontend for Paavalan's personal website, built using Next.js. The new design is optimized for responsiveness, performance, and accessibility, ensuring an improved user experience across various devices. Key updates include a more modern layout, enhanced styling, and simplified navigation to showcase Paavalan's portfolio and social presence.

**Changes Included:**

- Updated **homepage layout** with a new hero section, introducing Paavalan with a brief bio and professional summary.
- **Improved responsive design** for seamless viewing on mobile, tablet, and desktop.
- Added **prose styling for typography** using Tailwind CSS, enhancing readability in light and dark modes.
- Refactored **social links and profile image components** for better performance and alignment.
- Optimized **Next.js Image component usage** to ensure fast loading times and prioritized display.
- Updated styling with **CSS adjustments** for a modern look and feel, including hover effects and grayscale on the profile image.

**Screenshot**

![Screenshot 2024-10-28 at 01-30-59 Paavalan](https://github.com/user-attachments/assets/60792d30-20d5-471e-a513-deca2144b94a)

**How to Test:**

1. **Clone** this branch and install dependencies with `npm install` (or `yarn install`).
2. Run the development server with `npm run dev` (or `yarn dev`).
3. View the homepage at [http://localhost:3000](http://localhost:3000) and verify the following:
   - Responsive layout adjusts seamlessly on mobile, tablet, and desktop.
   - Profile image displays correctly with hover effects.
   - Links to social profiles open in a new tab and are functional.
   - Typography and layout are clean, and text is easy to read in both light and dark modes.

>**Notes:**
>
>This redesign serves as the foundation for further enhancements. Feedback on layout >adjustments or additional features is welcome.
